plugins/command-manager/src/main/java/com/wazuh/commandmanager/rest/RestPostCommandAction.java

```diff
@@ -94,42 +95,67 @@ private RestChannelConsumer handlePost(RestRequest request) throws IOException {
                 request.uri(),
                 request.getRequestId(),
                 request.header("Host"));
+
         // Get request details
+        if (!request.hasContent()) {
+            // Manejar el caso cuando no hay contenido
+            log.error("Body content is required");
+            return null;
+        }
         XContentParser parser = request.contentParser();
+        List<Command> commands = new ArrayList<>();
         ensureExpectedToken(XContentParser.Token.START_OBJECT, parser.nextToken(), parser);
+        parser.nextToken();
+        if (parser.nextToken() == XContentParser.Token.START_ARRAY) {
+            while (parser.nextToken() != XContentParser.Token.END_ARRAY) {
+                Command command = Command.parse(parser);
+                commands.add(command);
+            }
```

Comment:
The parsing from the request's body could be extracted to the model. SOmething along these lines:

```java
<List<Command> commands = Command.parseToArray(parser);
```